packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 46.13.0 - September 5, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-apex
+
+- Fix syntax highlighting for literal `when` values of String type in Apex switch statement ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #967](https://github.com/forcedotcom/salesforcedx-vscode/issues/967))
+
+- Fix syntax highlighting for block comments without spaces ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #921](https://github.com/forcedotcom/salesforcedx-vscode/issues/921))
+
+- Fix syntax highlighting for namespace names in Apex classes ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628))
+
+#### salesforcedx-vscode-core
+
+- Prevent `SFDX: Diff File Against Org` command from appearing when a non-SFDX project is opened ([PR #1608](https://github.com/forcedotcom/salesforcedx-vscode/pull/1608), [Issue #1600](https://github.com/forcedotcom/salesforcedx-vscode/issues/1600))
```

Comment:
@brpowell , could we rephrase this as:
Fix `SFDX: Diff File Against Org` to appear in the Command Palette only when opening SFDX projects
Or it could be:
Prevent `SFDX: Diff File Against Org` from appearing in the Command Palette when opening non-SFDX projects.